src/node/markdown/markdown.ts

```diff
@@ -48,7 +49,7 @@ export interface MarkdownRenderer extends MarkdownIt {
 export type { Header }
 
 export const createMarkdownRenderer = (
-  srcDir: string,
+  siteConfig: SiteConfig,
```

Comment:
`createMarkdownRenderer` is in fact an exported API used in some custom themes, so this would be a breaking change. I think we can move `cleanUrls` as an optional 3rd parameter instead.